### PR TITLE
PO-120 (Opencart<> Opencart Deprecated error)

### PR DESCRIPTION
**Pull request by rohitcbr**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
```
Comment (yashgit891) [inline .idea/.gitignore:2]: @rohitcbr what are these three files?

File: admin/language/en-gb/extension/payment/razorpay.php [hunk 2]
```diff
@@ -112,7 +112,7 @@
 $_['text_invoice']                                      = 'Invoice Details';
 $_['text_not_select_sub_ID']                             = 'Please Select Subscription ';
 $_['text_subscription_status']                          = 'Enable Subscription in Settings.';
-$_['text_webhook_cron_header']                          = 'Set the cron job in your WHMCS site server to call the Cron URL in every 5 mins frequncy.';
+$_['text_webhook_cron_header']                          = 'Set the cron job in your opencart site server to call the Cron URL in every 5 mins frequncy.';
```
Comment (yashgit891) [inline admin/language/en-gb/extension/payment/razorpay.php:115]: @rohitcbr please put platform as OpenCart.

File: .idea/.gitignore (hunk 1 above)
Comment (rohitcbr) [reply .idea/.gitignore:2]: @yashgit891 not sure of these files when pushing, should these 3 files be removed?

Comment (yashgit891) [reply .idea/.gitignore:2]: @rohitcbr if they are not part of sdk update and also not made by you then ideally it should not be present.

File: system/library/razorpay/razorpay-sdk/phpunit.xml.dist [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yashgit891) [inline system/library/razorpay/razorpay-sdk/phpunit.xml.dist:1]: @rohitcbr please remove this file.

File: system/library/razorpay/razorpay-sdk/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# This file is for unifying the coding style for different editors and IDEs
```
Comment (yashgit891) [inline system/library/razorpay/razorpay-sdk/.editorconfig:1]: @rohitcbr please remove this file.

File: system/library/razorpay/razorpay-sdk/.github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+name: feature.md
```
[the remark this answers is not in the document]
Comment (rohitcbr) [reply system/library/razorpay/razorpay-sdk/.github/ISSUE_TEMPLATE/feature.yml]: @yashgit891 removed in this commit https://github.com/razorpay/razorpay-opencart/pull/165/commits/94244e8ed3cc1cbb738076210f4125fe41409c8c

File: system/library/razorpay/razorpay-sdk/.editorconfig (hunk 4 above)
Comment (rohitcbr) [reply system/library/razorpay/razorpay-sdk/.editorconfig]: @yashgit891 removed in this commit https://github.com/razorpay/razorpay-opencart/pull/165/commits/94244e8ed3cc1cbb738076210f4125fe41409c8c

**Review by yashgit891 — APPROVED** [review]
LGTM

**Pull request merged**